CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Misc Improvements
 
+  * [#5161] (https://github.com/osmosis-labs/osmosis/pull/5161) Add test for GetMaximalNoSwapLPAmount
```

Comment:
```suggestion
```

Reply:
actually @ThanhNhann I am unable to make the change since this isn't coming from our repo. Can you make this change?

Reply:
Updated @czarcas7ic